### DISCUSSION TITLE
Poolday

### DISCUSSION THREAD

**Discussion by GT4nMore** [nzp-team/nzportable#915] [Map Releases]
Poolday from cs 1.6

Have not been been able to test on PSP!

Features:
Wall weapons
6 Perks
Doors
What not

Credits:
Squall for original cs1.6 map
Eoin for testing

Will update in the future, just make sure to leave a comment below if you have any suggestions/issues

Enjoy

[Download Here!](https://archive.org/details/poolday)

![nz_Poolday](https://github.com/user-attachments/assets/d69393d2-9ee7-45d9-be37-9b148d19dea2)

**Comment by GT4nMore** [comment]
Update 02-10-24

Moved Power to farthest room, now required to open both doors.

Closed doors to perks until power is activated.

Fixed issue with being able to dolphin dive out windows to out of bounds areas.

Minor optimizations.

Dogs stuck in walls should be fixed? (Tested as far as I'm aware it's fixed).

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard because the download is not hosted on archive.org

**Reply by GT4nMore** [reply]
Thanks mate, updated download link.

**Reply by mmkkiinn** [reply]
Create a psp version

**Reply by GT4nMore** [reply]
> Create a psp version

Hi, I just don't have anyone to test the PSP version, I'll give it a try on ppsspp but if you happen to have a psp dm me in discord if you can. Thanks!

**Reply by mmkkiinn** [reply]
Yes

**Reply by GT4nMore** [reply]
> Yes
I'm just a bit busy with exams at the moment, I'll try to release a PSP version by Sunday. Also found my old PSP so I'll get testing ;), Thanks for the suggestion mate <3

**Comment by irandiaz** [comment]
Hello, the map on my emulator has several problems that look all black.

**Comment by owenjames2023** [comment]
Hey I will be trying the map on psp for ya all. If you want any maps tried out just give message me

**Reply by katniny** [reply]
> I can have a PSP emulator so if it works on the emulator it will also work on the PSP I think

not necessarily true, but he's already confirmed:
> So it runs but the textures are glitched the rooms are black with little red and orange squares here and there

**Comment by owenjames2023** [comment]
So it runs but the textures are glitched the rooms are black with little red and orange squares here and there

**Comment by owenjames2023** [comment]
Do any of ya know what the problem is I test it on a psp 3000 running pro fast recovery 6.61

**Comment by owenjames2023** [comment]
Also I have the newest edition on nzp for my psp

**Comment by Jeyphr** [comment]
Here's my play through of this map: https://youtu.be/VCBzIe5BTT8

**Comment by GT4nMore** [comment]
Update 08-09-25

Apologies for updating this map so late.

Added new room where annoying block used to be.

Added chairs and tables to make the place more like the original.

Darkened most of the textures and opened up the roof and looks much better, thanks for the suggestion @Jeyphr ;).

Added back PPSH wallbuy.

Added 2 radios.

Shuffled perks around a little.

Dog stuck spots should be completely fixed.

Waypoints include special waypoints now and get rid of any invincible spots.



I am aware that the map does not work on PSPs and the like, my plan was to get nzp on PSP but never bothered. I will fix it within a week of this getting this uploaded but for now PC tested only. Also hard to know the issues if you dont have a working PSP lol so will try on emulator.

**Comment by Kingston4508** [comment]
map is very glitchy. made it to round 2 and died.

**Reply by s00240648** [reply]
Yo man, by any chance can you tell me what's wrong with the map. I only updated it recently so if there is 1 or 2 issues apologies. Thanks!

**Reply by GT4nMore** [reply]
oops wrong account lol

**Reply by Kingston4508** [reply]
map runs glitchy.  Z's sound is gugggggggg ahhhh gggg.  Its like the FPS is off.  Z's are not moving smooth.  The sounds is not smooth.  I'd be more than happy to play this map with anyone so they could see if its me or if its the map.   I'm in EST.

**Reply by GT4nMore** [reply]
whats your discord mate, feel free to add me on discord: gt4nmore

**Comment by Kingston4508** [comment]
@chicken-licken you have to try this one.

**Comment by Kingston4508** [comment]
Round 8.  awesome map.

**Comment by nonMat** [comment]
Fun!
